shell/src/main/java/alluxio/cli/fs/command/LoadCommand.java

```diff
@@ -144,6 +144,11 @@ private void runLoadTask(AlluxioURI filePath, URIStatus status, boolean local)
       }
       Protocol.OpenUfsBlockOptions openUfsBlockOptions =
           new InStreamOptions(status, options, conf, mFsContext).getOpenUfsBlockOptions(blockId);
+      if (openUfsBlockOptions.getNoCache()) {
+        // ignore "NO_CACHE" setting for "load"
+        openUfsBlockOptions = Protocol.OpenUfsBlockOptions.newBuilder(openUfsBlockOptions)
```

Comment:
we want to keep the original OpenUfsBlockOptions information but only update the NoCache field.